Source/GSXibLoading.m

```diff
@@ -466,6 +466,9 @@ - (NSString*) trigger
   return trigger;
 }
 
+#pragma clang diagnostic push
```

Comment:
I think all these pragma statements result in compiler warnings with gcc. In that case we now have three warnings instead of one. I need to test.

Reply:
A high hearty "Dammit" is in order... I'll just remove the pragmas. :/

` Compiling file GSXibLoader.m ...
 Compiling file GSXibLoading.m ...
GSXibLoading.m:469:0: warning: ignoring #pragma clang diagnostic [-Wunknown-pragmas]
 #pragma clang diagnostic push
 ^
GSXibLoading.m:470:0: warning: ignoring #pragma clang diagnostic [-Wunknown-pragmas]
 #pragma clang diagnostic ignored "-Wcast-of-sel-type"
 ^
GSXibLoading.m:511:0: warning: ignoring #pragma clang diagnostic [-Wunknown-pragmas]
 #pragma clang diagnostic pop
 ^
 Compiling file GSXibKeyedUnarchiver.m ...
 Compiling file GSXib5KeyedUnarchiver.m `

Reply:
fixed